File: C++/Fibonacci_Sequence.cpp
#include <iostream>
using namespace std;

void fib(int n)
{
 int t0=0,t1=1,s=0,i;
 
 if(n<=1) return ;
 
 for(i=2;i<=n;i++)
 {
    cout<<t1<<" ";
    s=t0+t1;
    t0=t1;
    t1=s;
 
 }
 
}

int main(){
    int n=7;
    fib(n);
    return 0; 
}

File: Fibonacci_Sequence.py
class Solution:
    def fib(self, n: int) -> int:
        a=0
        b=1
        c=0
        if n<=0 or n==1:
            return a
        else:
            Sequence=[]
            while c<n:
                Sequence.append(a)
                x=a+b
                a=b
                b=x
                c+=1
            return Sequence
